config/plugins/webhooks/gtn/script.js

```diff
@@ -0,0 +1,106 @@
+$(document).ready(function() {
+	var GTNView = Backbone.View.extend({
```

Comment:
I need to refresh my memory as to what my actual plan was for this, but assuming a global `Backbone` (or even `jQuery`/`$`) won't be robust long term.  Nothing to fix here, right now, because this will work -- but we'll want to revisit it.

Reply:
Easy for me to remove JQuery, and backbone is totally unnecessary here, happy to remove both and just write plain JS now if that's preferable?

Reply:
Plain old JS is definitely preferable, and is more future-proof, thanks!

Reply:
Updated: Removed backbone, jquery is a bit more difficult, so maybe leaving that until there's a long term vue fix?

Reply:
That's fine with me; as much as I want to get rid of it, we're still stuck with jquery for a while :)